deployment/ansible/group_vars/production.example

```diff
@@ -0,0 +1,241 @@
+---
+## GLOBAL APP SETTINGS
+# Which version of the app to install. Specifically, this designates a tag on quay.io that should be
+# used when pulling docker images. A list of valid tags can be found here:
+# https://quay.io/repository/azavea/driver-app?tag=latest&tab=tags
+# Generally, when setting up a new instance, you should use the highest-number tag available. The
+# 'latest' tag should be avoided when setting up new instances for production usage, because it may
+# contain bugs.
+app_version: "0.0.0"
+# The URL at which users will access the application. Do NOT include 'http' or 'https'; all DRIVER
+# instances are configured to use https.
+app_domain_name: "your.driver-domain.here"
+
+# The *internal* IP address of the hosts for the different components of the web application. DRIVER
+# is currently divided up to run on three servers: A database, a web app server, and a Celery
```

Comment:
`s/currently divided up to run on/runs on` -- I don't know how much the potential for future architecture revisions needs to be user-facing